config/services.yaml

```diff
@@ -108,8 +117,4 @@ services:
         arguments:
             - '@Gitlab\Client'
 
-    Buddy\Repman\Service\GitLabApi\RestGitLabApi:
-        arguments:
-            $url: '%env(APP_GITLAB_API_URL)%'
-
```

Comment:
I've only moved this to the top since I noticed that this is marked as a 'vendor' section